.github/workflows/docker-hub-release.yml

```diff
@@ -38,11 +45,21 @@ jobs:
           password: ${{ secrets.DOCKERHUB_TOKEN }}
 
       - name: Build image
+        id: push
         uses: docker/build-push-action@v5
         with:
           context: .
           file: ./Dockerfile
           platforms: ${{ env.PLATFORMS }}
           push: true
+          sbom: true
           tags: |
             ${{ env.DOCKER_NAMESPACE }}/draupnir:${{ env.RELEASE_VERSION }}
+
+      - name: Attest
+        uses: actions/attest-build-provenance@v1
+        id: attest
+        with:
+          subject-name: ${{ env.DOCKER_NAMESPACE }}/draupnir:${{ env.RELEASE_VERSION }}
+          subject-digest: ${{ steps.push.outputs.digest }}
+          push-to-registry: true
```

Comment:
how does the SBOM get made? does it take into account `yarn.lock`? Seems like https://github.com/anchore/syft gets [used by default](https://docs.docker.com/build/attestations/sbom/#sbom-generator) which is probably ok idk.

Reply:
Going to check that in a moment but it is supposed to do the sbom of the yarn stuff too. If not that would be worrying. It would be solvable by doing an sbom ourself and providing it to the docker image at `/usr/local/share/sbom/` but I would rather not need to do that.

Reply:
Yes it reads the lockfile:

```
> syft . -q -o json | jq '.artifacts | map(select(.locations | first | select(.path == "/yarn.lock"))) | length'
579
```

Reply:
that's cool